packages/frontend-2/components/project/page/models/Card.vue

```diff
@@ -178,10 +210,28 @@ const importArea = ref(
   }>
 )
 
+const actions = ref(
+  null as Nullable<{
+    showUploads: () => void
+  }>
+)
+
 const isVersionUploading = ref(false)
 const showActionsMenu = ref(false)
 const hovered = ref(false)
 
+const showLastUploadFailed = computed(() => {
+  if (isPendingModelFragment(props.model)) return false
+  const lastUpload = props.model.lastUpload?.items[0]
+  const lastVersion = props.model.lastVersion?.items[0]
+
+  // Only show if last upload failed & there is no last version,
+  // or last version is older than last upload
+  if (lastUpload?.convertedStatus !== FileUploadConvertedStatus.Error) return false
```

Comment:
If there was a stuck pending upload, we used to show it even if there were newer successfully created versions. This fixes that - if you create a new version after the pending upload, its gonna show that version, not the pending (stuck) upload.